DeploymentExperiences.md

```diff
@@ -1,8 +1,13 @@
 - Summarize the key steps in the deployment process. 
   - What went well?
+  I was able to deploy and get the app working after my 3rd try, there weren't too many bugs to figure out.
   - What challenges did you face? 
+  I initially had trouble getting the personal notes to work and had bugs regarding my migrations.
   - How far did you get?
+  I was able to complete the notes app and deploy it to Heroku.
 - We have utilized docs from several different projects over the course of this Sprint.
   - Which docs did you think were the most helpful? 
+    I think the DJ-Database-URL doc ended up being most useful
   - Name one or two specific things about this resource that you think sets it apart from other docs you have referenced in the past. 
 
+  This doc was useful because it allowed me to see what was needed for installation all the way to usage.
```

Comment:
Your finished project must include all of the following requirements:

* [ ] A link to your Djorg project repo
-- No link supplied
* [ ] A link to your live site, if you were able to deploy
-- No link supplied
* [ ] Completed summary of your deployment process in DeploymentExperiences.md
-- You have questions answered in a file called DeploymentExperience.md; however, it does not follow the guidelines set forth in the directions. An image was supplied with dummy text on what a summary ought to look like. We need a couple complete paragraphs that talk about the answers you have supplied in a professional writing style. Answering questions is a good starting point, but you need to write an actual essay regarding your experience. 

MVP not met.